src/components/SecondaryPanes/Frames/index.js

```diff
@@ -204,6 +204,7 @@ class Frames extends Component<Props, State> {
   }
 }
 
+// Should be converted to flow types... what is flow types ?!
```

Comment:
check other components for flow props 

https://flow.org/

Reply:
*thumbs up*